apps/dash-image-annotation/dash-bounding-box-image-annotation/app.py

```diff
@@ -0,0 +1,450 @@
+                ),
+            ],
+            className="seven columns",
+        ),
+        # Sidebar
+        html.Div(
+            id="sidebar",
+            children=[
+                html.Div(
+                    id="table-container",
+                    children=[
+                        # Timestamp table
+                        dash_table.DataTable(
+                            id="timestamp-table",
+                            columns=[{"name": "Timestamp", "id": "Timestamp"}],
+                            style_data={"height": 40},
+                        ),
+                        # Data table
+                        dash_table.DataTable(
+                            id="annotations-table",
+                            columns=[
+                                dict(
+                                    name=n,
+                                    id=n,
+                                    presentation=(
+                                        "dropdown" if n == "Type" else "input"
+                                    ),
+                                )
+                                for n in columns
+                            ],
+                            editable=True,
+                            style_data={"height": 40},
+                            dropdown={
+                                "Type": {
+                                    "options": [
+                                        {"label": o, "value": o}
+                                        for o in annotation_types
+                                    ],
+                                    "clearable": False,
+                                }
+                            },
+                            style_cell_conditional=[
+                                {"if": {"column_id": "Type"}, "textAlign": "left"}
+                            ],
+                        ),
+                    ],
+                ),
+                dcc.Store(id="graph-copy", data=fig),
+                dcc.Store(
+                    id="annotations-store",
+                    data=dict(
+                        **{filename: {"shapes": []} for filename in filelist},
+                        **{"starttime": time_passed()}
+                    ),
+                ),
+                dcc.Store(id="image_files", data={"files": filelist, "current": 0}),
+                html.H6("Type of annotation"),
+                dcc.Dropdown(
+                    id="annotation-type-dropdown",
+                    options=[{"label": t, "value": t} for t in annotation_types],
+                    value=DEFAULT_ATYPE,
+                    clearable=False,
+                ),
+                html.H6("Choose image"),
+                html.Button("Previous", id="previous", className="button"),
+                html.Button("Next", id="next", className="button"),
+                html.Button("Add Shape", id="add-shape", className="button"),
```

Comment:
do we still need this "add shape" button?

Reply:
Not really :upside_down_face: